src/target/target.c

```diff
@@ -3073,8 +3076,7 @@ static int handle_target(void *priv)
 			target_reset_examined(target);
 			retval = target_examine_one(target);
 			if (retval != ERROR_OK) {
-				LOG_TARGET_DEBUG(target, "Examination failed, GDB will be halted. "
-					"Polling again in %dms",
+				LOG_TARGET_DEBUG(target, "Examination failed. Polling again in %dms",
```

Comment:
Detail: 
```suggestion
				LOG_TARGET_DEBUG(target, "Examination failed. Polling again in %d ms",
```

Reply:
In the OpenOCD code that I did not write, I see 3 cases of "%d ms" and 3 cases of "%dms". I'm inclined to leave this as-is, and let upstream worry about it when they get around to nitpicking whitespace in debug messages.

Reply:
Agreed :)